extensions/interactions/GraphInput/graph-input-interactions.module.ts

```diff
@@ -22,18 +22,22 @@ import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { FormsModule } from '@angular/forms';
-import { SharedComponentsModule } from 'components/shared-component.module';
 import { GraphVizComponent } from './directives/graph-viz.component';
 import { InteractiveGraphInput } from './directives/oppia-interactive-graph-input.component';
 import { ResponseGraphInput } from './directives/oppia-response-graph-input.component';
 import { ShortResponseGraphInput } from './directives/oppia-short-response-graph-input.component';
+import { TranslateModule } from 'filters/translate.module';
+import { MaterialModule } from 'components/material.module';
+import { DirectivesModule } from 'components/../directives/directives.module';
```

Comment:
Why the path contains '..'?

Reply:
Oh yeah... I have been meaning to bring this up... for some reason, paths starting from /core/templates don't get resolved. Do you know why? The path here would ideally be core/templates/directives/directives.module but this errors out.

Reply:
Changed the import path by adding directive to path in tsconfig